docs/en/user/upgrade-2.0.x-2.1.0.rst

```diff
@@ -0,0 +1,50 @@
+Upgrading from 2.0.x to 2.1.0
+=============================
+
+.. warning::
+Before this migration, if you configured the Pocket import by adding your consumer key in Internal settings, please do a backup of it: you'll have to add it into the Config page after the upgrade.
+
+Upgrade on a dedicated web server
+---------------------------------
+
+The last release is published on https://www.wallabag.org/pages/download-wallabag.html. In order to upgrade your wallabag installation and get the last version, run the following commands in you wallabag folder (replace ``2.1.0`` by the last release number):
+
+::
+
+    git fetch origin
+    git fetch --tags
```

Comment:
`git fetch --all` should be enough, nope? (instead of `git fetch origin` + `git fetch --tags`)